tobac/analysis.py

```diff
@@ -563,6 +1044,56 @@ def area_histogram(
 def histogram_cellwise(
     Track, variable=None, bin_edges=None, quantity="max", density=False
 ):
+    """Create a histogram of the maximum, minimum or mean of
```

Comment:
I think on this function and the corresponding `featurewise` function, it would be helpful in the documentation to include what we mean by featurewise and by cellwise.